### PR TITLE
Refactor: move `formatted_birthday` to decorator

### PR DESCRIPTION
**Pull request by jessieay**
* Perhaps we should store view-specific methods like this in decorators?

cc @hartsick  @luigi

### CODE REVIEW

File: app/decorators/member_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class MemberDecorator
+  def initialize(member)
+    @member = member
+  end
+
+  delefate :birthday, to: member
```
Comment (luigi) [inline app/decorators/member_decorator.rb]: Typo. :)

File: app/decorators/member_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  def formatted_birthday
+    if birthday.present?
+      birthday.strftime("%m/%d/%Y")
+    else
+      ""
+    end
```
Comment (luigi) [inline app/decorators/member_decorator.rb]: I like `birthday.present? : birthday.strftime("%m/%d/%Y") : ""` but I don't know how the team/rubocop feel about it.

File: db/migrate/20171121231237_remove_birthday_from_medicaid_applications.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class RemoveBirthdayFromMedicaidApplications < ActiveRecord::Migration[5.1]
```
Comment (jessieay) [inline db/migrate/20171121231237_remove_birthday_from_medicaid_applications.rb]: ack! somehow forgot to add this file to my last PR -- sorry! moving too fast over here

File: app/decorators/member_decorator.rb (hunk 2 above)
Comment (jessieay) [reply app/decorators/member_decorator.rb]: I am fine with that, not my usual style but nothing wrong with it and rubocop does not complain. anyone else have an opinion?

**Pull request merged**